config-bootstrapper/src/integrationTest/resources/config-bootstrapper-test/3_entitytypesv2.conf

```diff
@@ -0,0 +1,43 @@
+version = 1
+name = entitytypesv2
+commands = [
+	{
+    upgrade : [
+      {
+        type : EntityTypeV2,
+        action : ADD,
+        data : {
+          entityTypes = [
+            {
+              "name": "SERVICE",
+              "attributeScope": "SERVICE",
+              "idAttributeKey": "id",
+              "nameAttributeKey": "name"
+            },
+            {
+              "name": "API",
+              "attributeScope": "API",
+              "idAttributeKey": "id",
+              "nameAttributeKey": "name"
+            },
+            {
+              "name": "BACKEND",
```

Comment:
Is there a case where name != attributeScope or can that be implicit?

Reply:
Discussed this on the entity-service PR too. As of now they're same for different entities but later there could be cases so keeping the `name` as an explicit field.